sdk/tests/js/tests/out-of-bounds-test.js

```diff
@@ -136,11 +136,15 @@ var runDrawElementsTest = function(callTemplate, gl, wtu, ext) {
     gl.bufferData(gl.ARRAY_BUFFER, new Float32Array([ 0,0.5,0, -0.5,-0.5,0, 0.5,-0.5,0 ]), gl.STATIC_DRAW);
     gl.vertexAttribPointer(0, 3, gl.FLOAT, false, 0, 0);
 
-    var indexObject = gl.createBuffer();
 
+    debug('');
+    debug('Test null index buffer');
+    gl.bindBuffer(gl.ELEMENT_ARRAY_BUFFER, null);
+    wtu.shouldGenerateGLError(gl, gl.INVALID_OPERATION, wtu.replaceParams(callTemplate, {count: 0, type: 'gl.UNSIGNED_BYTE', offset: 0}));
```

Comment:
I didn't remember us deciding to make this specific change for zero-length draw calls. Is this needed because this is part of the validation done for draw calls with length > 0? If so, OK – what are we going to do about older versions of the conformance suite? Can you supply changes back to version 1.0.1 that allow both the old and new behaviors?

Reply:
Validation should be consistent [0,N], instead of special-casing 0. If the validation code usually requires an index buffer, then we should check it even if it's technically fine to omit. (particularly because there's no performance incentive to make no-op draw calls fast)

Reply:
OK.